src/forms/FormControl.scss

```diff
@@ -0,0 +1,366 @@
+// stylelint-disable primer/typography, primer/borders, primer/spacing, primer/box-shadow, max-nesting-depth
+
+:root {
+  accent-color: var(--color-accent-fg);
```

Comment:
This seems a bit general. Should we focus it to the selector using it?

Reply:
I think it only applies to a few `input` elements by default https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color

BUT, I removed it anyways. We're adding custom Checkbox/Radio styles instead of relying on [browser defaults](https://github.com/github/primer/issues/412) going forward.

Reply:
If we wanted to add it for all "native" controls, maybe [`native.scss`](https://github.com/primer/css/blob/46fb49cb9251c63b3db4639737b469286730381c/src/color-modes/native.scss) would be a good place. It's where we have the `color-scheme` and Windows High Contrast mode styles.